apps/system/lockscreen/js/lockscreen_state_manager.js

```diff
@@ -710,8 +710,8 @@
   /**
    * To check if one or multiple types of state is acceptable.
    *
-   * @param acceptables {[string]} - all state types that can be accepted.
-   * @param target {string|[string]} - the type(s) need to check
+   * @param acceptables {string} - all state types that can be accepted.
+   * @param target {string} - the type(s) need to check
```

Comment:
please trace code and see if it support both string/array, if so, the correct type will be `{string|string[]}`